tests/e2e/performance/test_pod_reattachtime.py

```diff
@@ -305,6 +308,8 @@ def test_pod_reattach_time_performance(
         )
 
         full_results.add_key("storageclass", self.sc)
+        full_results.add_key("copies_number", copies)
+        full_results.add_key("files_number_average", files_written_average)
```

Comment:
i think it will be better to add the `data_written` value as well

Reply:
Added it.